index.html

```diff
@@ -231,6 +231,30 @@ <h3>Languages</h3>
         <img src="assets/vector.svg " alt="">
       </div>
   </section>
+
+  <section class="contact-me">
+    
+    <form>
+      <div class="background">
+      <h2>contact-me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+      <label for="">
+        <input type="text" id="message" placeholder="Lorem Ipsum"/>
+      </label>
+      <label for="">
+        <input type="text" id="message" placeholder="loremipsum@gmail.com"/>
+      </label>
+      <textarea id="message-box" placeholder = "">
+        
+      </textarea>
+      <button class="el-botones">
+        <p>Get in touch</p>
```

Comment:
[Optional] Great job on your implementation! 👍 I noticed that you have included a `<p>` tag inside the `<button>` tag. While this is allowed, it's not necessary in this case. Since you can style the button directly, you can remove the `<p>` tag and apply the desired styles directly to the button element.

Removing the unnecessary `<p>` tag will simplify your HTML structure and make it more semantically correct. It's always a good practice to keep the markup as clean and minimal as possible.

Overall, your implementation is looking good! Feel free to make this adjustment to optimize your code further. Keep up the great work! 🌟